backend/question-service/server/app/routers/admin.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import HTTPException, Query, APIRouter
+from typing import Optional
+import sys
+
+sys.path.append("..")
+from core.database import collection
+from utils.jsend import JSendStatus, jsend_response
+
+
+admin_router = APIRouter(
+    prefix="/admin",
+    tags=["Admin"],
+)
+
+
+@admin_router.post("/questions")
+async def create_question(question: dict):
+    """Creates a new question in the database"""
+
+    # Check if id is provided. If not, auto-increment from the last id
+    if "id" not in question:
+        last_question = collection.find_one(sort=[("id", -1)])
+        if last_question:
+            question["id"] = last_question["id"] + 1
+        else:
+            question["id"] = 1
+
+    # Check if question with the same id already exists
+    if collection.find_one({"id": question["id"]}):
+        raise HTTPException(
+            status_code=400, detail="Question with the same id already exists.")
+    result = collection.insert_one(question)
+
+    question["_id"] = str(question["_id"])
```

Comment:
What's this line for?

Reply:
MongoDB's `_id` is type ObjectId and is not readable by python so we need to convert it to string before we parse the data